packages/gnars-contracts/hardhat.config.ts

```diff
@@ -55,7 +62,7 @@ const config: HardhatUserConfig = {
   },
   etherscan: {
     apiKey: {
-      mainnet: process.env.ETHERSCAN_API_KEY!,
```

Comment:
probably fine to leave this line since the "mainnet" network is still defined above